public/Add-DbaReplArticle.ps1

```diff
@@ -0,0 +1,179 @@
+function Add-DbaReplArticle {
+    <#
+    .SYNOPSIS
+        Adds an article to a publication for the database on the target SQL instances.
+
+    .DESCRIPTION
+        Adds an article to a publication for the database on the target SQL instances.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Accepts PowerShell credentials (Get-Credential).
+
+        Windows Authentication, SQL Server Authentication, Active Directory - Password, and Active Directory - Integrated are all supported.
+
+        For MFA support, please use Connect-DbaInstance.
+
+    .PARAMETER Database
+        The database on the publisher that contains the article to be replicated.
+
+    .PARAMETER Publication
+        The name of the replication publication.
+
+    .PARAMETER Schema
+        The schema name that contains the object to add as an article.
+        Default is dbo.
+
+    .PARAMETER Name
+        The name of the object to add as an article.
+
+    .PARAMETER Filter
+        Horizontal filter for replication, implemented as a where clause, but don't include the word WHERE>
```

Comment:
Is the `>` needed?

Reply:
```suggestion
        Sets the where clause used to filter the article horizontally, e.g., DiscontinuedDate IS NULL
```
A bit easier to process (considering I haven't touched replication in a bit) and even reading the doc meaning the WHERE keyword I think will throw the beginners off a bit.
Ref example: https://learn.microsoft.com/en-us/dotnet/api/microsoft.sqlserver.replication.transarticle.filterclause?view=sqlserver-2016#microsoft-sqlserver-replication-transarticle-filterclause